README.md

```diff
@@ -1,56 +1,59 @@
 killbill-client-php
 ===================
 
-PHP client library for killbill
+PHP client library for [Killbill](http://killbill.io)
 
 Configuration
 -------------
 
 In order to user the library, you need to:
 
-1. Download the library and unzip it in your application
```

Comment:
I'm not sure about this. What composer does is to create an autoload file, which registers an autload function. This could probably be done manually. However, this wouldn't make too much sense.

It is really easy to use composer in a non-composer project. Just use composer to download the dependency and use it's auto-generated autoload script. The complete `vendor/` folder can even be checked into version control, so that no additional step is required while deploying.

What do you think @pierre?